frame/dvm/ethereum/src/account_basic.rs

```diff
@@ -0,0 +1,74 @@
+use frame_support::traits::Currency;
+use pallet_evm::{Account as EVMAccount, AccountBasicMapping, AddressMapping};
+use sp_core::{H160, U256};
+use sp_runtime::traits::{UniqueSaturatedFrom, UniqueSaturatedInto};
+
+pub struct DVMAccountBasicMapping<T>(sp_std::marker::PhantomData<T>);
+
+impl<T: crate::Trait + darwinia_balances::Trait<darwinia_balances::Instance0>> AccountBasicMapping
+	for DVMAccountBasicMapping<T>
+{
+	/// Get the account basic in EVM format.
+	fn account_basic(address: &H160) -> EVMAccount {
+		let account_id = <T as pallet_evm::Trait>::AddressMapping::into_account_id(*address);
+		let nonce = frame_system::Module::<T>::account_nonce(&account_id);
+		let helper = U256::from(10).overflowing_pow(U256::from(9)).0;
+
+		// Get balance from T::Currency
+		let balance = T::Currency::free_balance(&account_id);
+		let balance = U256::from(UniqueSaturatedInto::<u128>::unique_saturated_into(balance));
```

Comment:
Why not just `saturated_into`?

Reply:
The EVM balance value represents as `U256` at the end. So I do the conversation that  u128 to u256 here for later calculations.

Reply:
But `let _: U256 = balance.saturated_into();`

Reply:
Let me have a try!

Reply:
Done.